views/user/email-confirmation.ejs

```diff
@@ -0,0 +1,20 @@
+<!DocType html>
+<html>
+    <head>
+        <% include ../template/head.ejs %>
+        <title>Verification required</title>
+    </head>
+    <body>
+        <% include ../template/nav.ejs %>
+
+        <div class="container body-layout ">
+            <div class="img-center-holder margin-dtop">
+                <img class="sponsor-img" src="img/icon.jpg" alt="Corgi">
+                <h2> A verification email has been sent to your email. Please verify your account, thank you! </h2>
```

Comment:
No need to add spaces in the front and at the end of this sentence

Reply:
Also, it would be good to change the color of icon.jpg from white to clear.

Reply:
Ok, the image can be edited (delete the background) using Photoshop or similar editing software and replace the old one. Do it if you can or Jiazhi (our board member) can take care of it